src/support_sphere/lib/presentation/components/auth/login_form.dart

```diff
@@ -87,11 +112,14 @@ class _PasswordInput extends StatelessWidget {
           onChanged: (password) =>
               context.read<LoginCubit>().passwordChanged(password),
           obscureText: !state.showPassword,
+          autovalidateMode: AutovalidateMode.onUserInteraction,
+          validator: (value) => validateValue([
+            FormBuilderValidators.required(),
+            FormBuilderValidators.password(),
+          ], value, context),
```

Comment:
Broad question: _**should**_ we be validating a user's password here? We definitely should when they sign up with it or change it, but when I tried it out in the web view, having this validation run while I was typing in the password felt kind of weird -- like if someone was watching over my shoulder, they could know at what point in my password a number occurs, for example.

I think there's some validation that can happen here for sure ([length](https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html#implement-proper-password-strength-controls) for example), but if a password doesn't meet our signup/password change requirements, it'll cause the login attempt to fail.

(I suppose the same could be said for the email validation too)

Reply:
Thinking about this. You're totally right on the password validation step. Though in terms of email, I think having at least the correct email validation.. say someone having an `@` would be valuable to save on network traffic of people sending in junk email address that is not valid. I'll change that. Thanks for thinking about that! I'll just leave the password validation off.